x-pack/plugins/ml/public/application/jobs/jobs_list/components/delete_job_modal/delete_job_modal.tsx

```diff
@@ -150,6 +156,21 @@ export const DeleteJobModal: FC<Props> = ({ setShowFunction, unsetShowFunction,
                     onChange={(e) => setDeleteUserAnnotations(e.target.checked)}
                     data-test-subj="mlDeleteJobConfirmModalDeleteAnnotationsSwitch"
                   />
+                  {hasAlertingRules ? (
+                    <>
+                      <EuiSpacer size={'s'} />
+                      <EuiSwitch
+                        label={i18n.translate(
+                          'xpack.ml.jobsList.resetJobModal.deleteAlertingRules',
+                          {
+                            defaultMessage: 'Delete alerting rules',
```

Comment:
Noticed that the `Delete annotations` switch label ends with a `.`.  The format you have here, without the `.` is consistent with what we use elsewhere, so could you remove the `.` from the `Delete annotations.` label?

Reply:
Updated in https://github.com/elastic/kibana/pull/176049/commits/4797ce56d941e0d431a5a6aee4475ccba1b1909b